pkg/apis/componentconfig/types.go

```diff
@@ -62,3 +62,213 @@ const (
 	ProxyModeUserspace ProxyMode = "userspace"
 	ProxyModeIPTables  ProxyMode = "iptables"
 )
+
+type KubeletConfiguration struct {
```

Comment:
One side effect of moving this out of options.go is that it is no longer close to where the defaults are defined.  Not sure if there is anything that can be done about this.